templates/index.html

```diff
@@ -39,7 +39,7 @@
             <span class="navbar-text">
             </span>
         </div>
-    </nav>
+    </nav>-->
     <div style='height: 100vh;' class="jumbotron">
         <h1 class="display-4">Hello, gamer!</h1>
         <p class="lead">Welcome to the abode of multiplayer gaming!</p>
```

Comment:
Please add the logout functionality to the exit button on this page. Also, while you're at it, rename it to 'Log out'

Reply:
I pushed the branch